### PR TITLE
Improve the compare page (SDSC ORD Hackathon)

### PR DESCRIPTION
**Pull request by faymanns**
- Searchable drop down menus
- Details tabs on the side connected to the plot
- Axis labels + tick text
- Fix zoom of plot
- Linear regression line + Pearson R + R^2 + p-value
- Download buttons

before:
Screenshot 2024-10-28 at 12-07-39 Brain-Score

after:
Screenshot 2024-10-28 at 12-07-08 Brain-Score

### CODE REVIEW

File: benchmarks/templates/benchmarks/compare.html [hunk 1]
```diff
@@ -2,33 +2,107 @@
...
+                    <a href="https://www.biorxiv.org/content/10.1101/407007">Schrimpf & Kubilius et al. 2018</a> with
+                    ImageNet).
+                    This connection between machine learning and neuroscience desiderata implies a striking
+                    synergy between research in these two domains, as building better-performing models might also
+                    lead to better models of the brain.
+                </div>
+            </div>
+            <div class="comparison_selector"
+                 data-benchmark-x="FreemanZiemba2013.V1-pls_v2" data-benchmark-y="ImageNet-C-top1_v1">
+                <p class="" data-identifier="comparison_selector-V1_robustness">
```
Comment (KartikP) [inline benchmarks/templates/benchmarks/compare.html:75]: `ImageNet-C-top1_v1` does not appear to be auto-selected when "Alignment to early visual cortex V1 is correlated with model robustness" is selected.

image

**Review by KartikP — COMMENTED** [review]
LGTM for the most part but please confirm that the comment replicates the error on your end. It does not work for me.

File: benchmarks/templates/benchmarks/compare.html (hunk 1 above)
Comment (mschrimpf) [reply benchmarks/templates/benchmarks/compare.html:75]: Odd, this does show up on my end. Should we still merge or do you want to comment that button out for now?

**Review by KartikP — APPROVED** [review]
LGTM! Will check on deployment if all buttons change the compare view.

**Pull request merged**